db/schema.rb

```diff
@@ -11,33 +11,37 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160708192840) do
+ActiveRecord::Schema.define(version: 20170105165853) do
 
   create_table "bus_stops", force: :cascade do |t|
-    t.string  "name",                      limit: 255, default: "", null: false
-    t.string  "hastus_id",                 limit: 255,              null: false
-    t.string  "accessible",                limit: 255
-    t.string  "bench",                     limit: 255
-    t.string  "curb_cut",                  limit: 255
-    t.string  "lighting",                  limit: 255
-    t.string  "mounting",                  limit: 255
-    t.string  "mounting_direction",        limit: 255
-    t.string  "schedule_holder",           limit: 255
-    t.string  "shelter",                   limit: 255
-    t.string  "sidewalk",                  limit: 255
-    t.string  "sign",                      limit: 255
-    t.string  "trash",                     limit: 255
-    t.boolean "bolt_on_base"
-    t.boolean "bus_pull_out_exists"
-    t.boolean "extend_pole"
-    t.boolean "has_power"
-    t.boolean "new_anchor"
-    t.boolean "new_pole"
-    t.boolean "solar_lighting"
-    t.boolean "straighten_pole"
-    t.boolean "system_map_exists"
-    t.integer "mounting_clearance_after",  limit: 4
-    t.integer "mounting_clearance_before", limit: 4
+    t.string   "name",                      limit: 255, default: "", null: false
+    t.string   "hastus_id",                 limit: 255,              null: false
+    t.string   "accessible",                limit: 255
+    t.string   "bench",                     limit: 255
+    t.string   "curb_cut",                  limit: 255
+    t.string   "lighting",                  limit: 255
+    t.string   "mounting",                  limit: 255
+    t.string   "mounting_direction",        limit: 255
+    t.string   "schedule_holder",           limit: 255
+    t.string   "shelter",                   limit: 255
+    t.string   "sidewalk",                  limit: 255
+    t.string   "sign",                      limit: 255
+    t.string   "trash",                     limit: 255
+    t.boolean  "bolt_on_base"
+    t.boolean  "bus_pull_out_exists"
+    t.boolean  "extend_pole"
+    t.boolean  "has_power"
+    t.boolean  "new_anchor"
+    t.boolean  "new_pole"
+    t.boolean  "solar_lighting"
+    t.boolean  "straighten_pole"
+    t.boolean  "system_map_exists"
+    t.integer  "mounting_clearance_after",  limit: 4
+    t.integer  "mounting_clearance_before", limit: 4
```

Comment:
Everything upwards of here is not a change. The `limit` values were automatically pushed over by one character (apparently).